home/views.py

```
@@ -71,7 +71,7 @@ class CustomPasswordResetCompleteView(PasswordResetCompleteView):
 #! ---------------------------
 
 
-def home(request):
+def home(reqest):
     user, type = request.user, request.user.get_user_type_display()
```

Ruff (0.11.9)

Local variable `user` is assigned to but never used

Remove assignment to unused variable `user`

(F841)